### ISSUE TITLE
Add support for basic html tags

### ISSUE THREAD

**Issue by dcastro** [serokell/xrefcheck#93]
# Clarification and motivation

HTML tags can be used in a markdown file.

For example, in our own readme, we have a `<img>` tag:

```
<img src="https://user-images.githubusercontent.com/5394217/70820564-06b06e00-1dea-11ea-9680-27f661ca2a58.png" alt="Output sample" width="600"/>
```

We should add support for basic HTML tags (embedded in markdown files), such as:
* anchors: `<a href="link">Text</a>`
* images: `<img src="link">` 

We should investigate which other tags we want to support.

# Acceptance criteria

`xrefcheck` verifies whether anchors and image tags (and possibly others) are valid.

**Comment by Martoon-00** [comment]
Oh, that's an interesting point.

Checking images would indeed be helpful, if we succeed to implement #92.

Anchors are handled at the moment, but it would be really nice to see which other HTML tags can make sense in the documentation. E.g. I often use spoilers, but here we probably don't have anything to verify...

**Comment by dcastro** [comment]
> Anchors are handled at the moment

I don't think we handle anchor tags at the moment. We do handle `<a name="some-name">` in headers, but that's it.

From `tests/markdowns/without-annotations/anchors_in_headers.md`:
```
## <a name="stuff-section"></a> Some stuff

[My link](#stuff-section)
```

**Comment by dcastro** [comment]
On that point, we should also support for _target locations_ (i.e. `<a name="loc">`) outside of headers too.

**Comment by Martoon-00** [comment]
@Sereja313 now switched to another project, he likely won't be able to work on this task.

@Sereja313 if you have begun working on it and have something to share, or if you suddenly come to implement this issue for some personal preference, I dunno, please mention this here.

**Comment by Sereja313** [comment]
@Martoon-00 I'm going to finish the PR for this issue this week.

**Comment by Martoon-00** [comment]
Ah, great then :+1:

**Comment by Martoon-00** [comment]
@Sereja313 How is it going?

If you see that you completely lack time, feel free to just leave a WIP PR where part of the work is done.

**Comment by Sereja313** [comment]
Yes, I've been a bit busy, sorry... I'll have some free time tomorrow.

**Comment by Martoon-00** [comment]
Great, thanks for posting the PR :+1: 

I think we will take it from now on (unless you'd like to follow the PR until it's merged).
